src/components/Organization/Organization.scss

```diff
@@ -3,6 +3,10 @@
 
 .organization {
   padding: 1.5em;
+
+  &__title > h4 {
+    margin-top: 0; // Overrides markdown scss
```

Comment:
Why add `markdown` in the first place?

Reply:
Seems all the basic styles are now scoped under `.markdown`where they previously existed under `page__content` (now removed).

I'm not happy with this solution, but I'm not going to redo the CSS architecture as well when my focus is fixing broken links

Reply:
Me neither, will fix :)